Gaijinizer patch/Map122.txt

```diff
@@ -721,9 +747,10 @@ original size.#MTLed
 
  あの連中も、下手な実験はできなかったらしい。
 > CONTEXT: Map122/events/5/pages/1/336/Dialogue/enemy14
-And not completely the same, I wanted to be more
-robust and better. Those guys also couldn't do
-bad experiments.#MTLed
+And it wouldn't return the same, it would
+apparently reform into something stronger,
+better. So those guys had to be careful how they
+experimented on me.#Ron
 > END STRING
```

Comment:
rewording:
>_**It wouldn't even be the same body...
It would be stronger and more powerful.
So they had to be careful what experiments
they did on me.**_